src/lib/menu/config.ts

```diff
@@ -21,8 +21,8 @@ export const MENU_ID = Object.freeze({
     ADMINISTRATION_USER: 'administration.user',
     ADMINISTRATION_ROLE: 'administration.role',
     ADMINISTRATION_POLICY: 'administration.policy',
-    ADMINISTRATION_SETTINGS: 'administration.settings',
-    ADMINISTRATION_PROVIDER: 'administration.provider',
+    // ADMINISTRATION_SETTINGS: 'administration.settings',
+    // ADMINISTRATION_PROVIDER: 'administration.provider',
```

Comment:
Is these changes are needed? 👀

Reply:
It's not necessary, but it's a service that doesn't exist yet, so this config can be used anywhere, so I annotated it!